capstone/fitness/src/main/java/com/google/sps/fit/Exercise.java

```diff
@@ -113,82 +117,85 @@ public Exercise build() {
   private Exercise(){}
 
   /**
-   * Returns true if average of Exercise values are better than the given Exercise's average 
-   * values in each type.
-   * Note: Uses average to ignore set count.
+   * Returns true if Exercise's values are better than or overall equal to the given Exercise's 
+   * values in each type. Uses average if values should decrement, uses summation if not. The equal
```

Comment:
Not blocking for this PR, but it doesn't seem immediately obvious why decreasing values should be average but the opposite should be summed. Maybe this can be something put into SetType for the future.

Reply:
Yeah my bad, the math just kinda seemed to work out for the example edge cases I gave myself. For example, if I was comparing a 600 sec duration to two 300 sec durations, I would say the two 300s are better in the decrement case because they are smaller numbers. However, if I use summation then the algorithm would claim they are equal even though doing one set of 600 sec for something would be significantly easier than two sets of 300 secs (e.g. Running). I'll probably move this logic to SetType though because it does seem to clutter the functions a lot.

**TL;DR** - The math just kinda works (see design doc for examples), but I'll eventually move the logic to SetType someday.